python/cudf/cudf/tests/test_replace.py

```diff
@@ -1371,3 +1371,17 @@ def test_replace_nulls(gsr, old, new, expected):
         expected.sort_values().reset_index(drop=True),
         actual.sort_values().reset_index(drop=True),
     )
+
+
+def test_fillna_columns_multiIndex():
```

Comment:
We're inconsistent about naming but we have more instances of `multiindex` than `multiIndex` in the test suite. Let's move towards using `multiindex`.
```suggestion
def test_fillna_columns_multiindex():
```